<?xml version="1.0" ?>
<testCase duration="0.062" fail="1" name="SET-250" pass="0" startdate="20160216" starttime="125431">
	<block fail="0" pass="0"/>
	<block fail="0" pass="0"/>
	<block fail="1" pass="0">
		<block fail="1" pass="0">
			<call fail="0" pass="1">functions.addValues(vars.a, &quot;A&quot;)</call>
			<call fail="1" pass="0">functions.raiseError('Intentional exception in execution phase')</call>
			<call fail="0" pass="0">functions.raiseError('Should not run')</call>
		</block>
		<block fail="0" pass="1">
			<call fail="0" pass="1">functions.addValues(vars.a, &quot;B&quot;)</call>
		</block>
	</block>
	<block fail="0" pass="1">
		<call fail="0" pass="1">functions.compareValues(vars.a, &quot;AB&quot;)</call>
	</block>
</testCase>
